src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -2651,6 +2651,11 @@ public sealed override BoundNode VisitDeconstructionLocalPendingInference(Decons
         {
             throw ExceptionUtilities.Unreachable;
         }
+
+        public override BoundNode VisitVoid(BoundVoid node)
```

Comment:
PreciseAbstractFlowPass should handle all nodes. One of my tests ran into an assert related to BoundVoid.